src/alfasim_sdk/_internal/constants.py

```diff
@@ -465,3 +465,15 @@ class FlowPatternModel(Enum):
 
 
 MULTI_INPUT_TYPE_SUFFIX = "_input_type"
+
+
+class CorrelationsOilViscosity(Enum):
```

Comment:
Those enums are not used in case description? (or alfacase?)

Reply:
not as of now